modules/gdscript/gdscript_parser.cpp

```diff
@@ -41,6 +41,7 @@
 #include "core/io/resource_loader.h"
 #include "core/math/math_defs.h"
 #include "scene/main/multiplayer_api.h"
+#include "modules/regex/regex.h"
```

Comment:
As explained in rocketchat this module is not obligatory, you need to handle cases where it is not available

Reply:
in case its not available, the regex from c++ can be used? because if not, how I'm supposed to do?

Reply:
The standard library is not allowed, see [here](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html), (as also explained)

You should probably process the text directly instead of relying on RegEx